### PR TITLE
[#1690] Add some logs to ltag, trying to identify what goes wrong

### PR DESCRIPTION
**Pull request by olaurendeau**
Debug for #1690

### CODE REVIEW

**Comment by asaunier** [comment]
Could you please rather push your branch to the main repo (not to your fork) so that a docker image is built and made available at https://hub.docker.com/r/c2corg/v6_ui/tags/
I could then use the image on the demo server.

But I think you have to fix the formatting errors first else I don't think Travis will create the docker image:
>c2corg_ui/format/ltag.py:40:80: E501 line too long (80 > 79 characters)
c2corg_ui/format/ltag.py:110:80: E501 line too long (193 > 79 characters)

File: c2corg_ui/format/ltag.py [hunk 1]
```diff
@@ -107,7 +107,17 @@ def test(self, parent, block):
         is_ltag = self.is_ltag(block)
         rows = block.split('\n')
 
-        log.debug("This block " + ("has ltags " + ("but will be skiped" if self.skip else "and will be processed") if is_ltag else "has no ltags") + " and is starting with \"" + rows[0] + "\"")
+        log_message = "This block "
+        if is_ltag:
+            log_message += "has ltags "
+            if self.skip:
+                log_message += "but will be skiped"
+            else
```
Comment (asaunier) [inline c2corg_ui/format/ltag.py]: ``else:``

**Comment by olaurendeau** [comment]
@asaunier I think I've figure it out.
This is because the `skip` variable isn't reseted between each document https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/format/ltag.py#L83
I try it and update the PR.

**Comment by olaurendeau** [comment]
Fixed by https://github.com/c2corg/v6_ui/pull/1695

**Pull request closed without merging**